assets/js/app.js

```diff
@@ -32,6 +32,23 @@ let csrfToken = document
 
 let Hooks = {};
 
+Hooks.AutoResizeTextarea = {
```

Comment:
Please remove this code. 

Sorry, for not having made that clearer: 
- We make the text field 3 rows higher in the desktop. 
- We leave the mobile version as it is. 
- We deactivate the possibility to resize the text field. 
- We do not add any JavaScript magic to auto enlarge it.